mix.lock

```diff
@@ -5,7 +5,7 @@
   "earmark": {:hex, :earmark, "1.3.5", "0db71c8290b5bc81cb0101a2a507a76dca659513984d683119ee722828b424f6", [:mix], [], "hexpm"},
   "ex_doc": {:hex, :ex_doc, "0.21.1", "5ac36660846967cd869255f4426467a11672fec3d8db602c429425ce5b613b90", [:mix], [{:earmark, "~> 1.3", [hex: :earmark, repo: "hexpm", optional: false]}, {:makeup_elixir, "~> 0.14", [hex: :makeup_elixir, repo: "hexpm", optional: false]}], "hexpm"},
   "gen_stage": {:hex, :gen_stage, "0.14.2", "6a2a578a510c5bfca8a45e6b27552f613b41cf584b58210f017088d3d17d0b14", [:mix], [], "hexpm"},
-  "google_api_pub_sub": {:hex, :google_api_pub_sub, "0.11.0", "ed8d65d28ce7fb2bf05ffb97ba678ab81dc1e3fe696c34bf2ff20f3a385dd61f", [:mix], [{:google_gax, "~> 0.2", [hex: :google_gax, repo: "hexpm", optional: false]}], "hexpm"},
+  "google_api_pub_sub": {:hex, :google_api_pub_sub, "0.14.0", "6babac64ac8ee2e705a180ad2c1ac15a58e68cbeee506aaa2e96f55b6aaa06da", [:mix], [{:google_gax, "~> 0.2", [hex: :google_gax, repo: "hexpm", optional: false]}], "hexpm"},
```

Comment:
no problem in bumping this, but there doesn't seem to be any meaningful changes:

https://diff.jola.dev/diff/google_api_pub_sub/0.11.0/0.14.0

if there was a regression with regards to json handling, perhaps it was in google_gax or tesla deps?

Reply:
@wojtekmach I've been looking, but I can't find anything significant.